doc/development/powershell.md

```diff
@@ -0,0 +1,165 @@
+This page contains guidelines for developing or updating powershell scripts used by the Azure SDK engineering system.
+
+Table of Contents
+=================
+
+* [TLDR](#tldr)
+* [Structure](#structure)
+* [Functionality](#functionality)
+* [Style](#style)
+* [Testing](#testing)
+   * [Unit Testing](#unit-testing)
+      * [Running Pester Tests](#running-pester-tests)
+   * [Local/Pipeline Functional Testing](#localpipeline-functional-testing)
+
+## TLDR
+
+_(i.e. too long, didn't read)_
+
+**DO**
+
+- Try to write scripts that can be run locally
+- Write unit tests for your scripts
```

Comment:
Nit: do you want to mention `pester` here? Maybe as a sub-point? If wasn't aware of it, I would think 

> Unit testing? I guess I'll write something that works like a test.

Reply:
Let me add a link to the Pester section!